sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/operations/_container_registry_operations.py

```diff
@@ -118,13 +118,13 @@ def get_repositories(
         def prepare_request(next_link=None):
             # Construct headers
             header_parameters = {}  # type: Dict[str, Any]
-            header_parameters['Accept'] = self._serialize.header("accept", accept, 'str')
+            header_parameters['Accept'] = self._client._serialize.header("accept", accept, 'str')
```

Comment:
I can't tell if you regenerated or if this is edited generated code. Ignore if former, if latter I would add a FIXME comment to flag these edits